sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -1015,8 +1015,8 @@ def sdc_pandas_dataframe_drop_impl(df, labels=None, axis=0, index=None, columns=
             break
 
     for column_id, column_name in enumerate(saved_df_columns):
-        func_text.append(f'new_col_{column_id}_data_{"df"} = get_dataframe_data({"df"}, '
-                         f'{df_columns_indx[column_name]})')
+        func_text.append(f'new_col_{column_id}_data_{"df"} = {"df"}._data['
```

Comment:
Curved brackets aren't necessary here, since we use quoted str in them anyway. It's the same as:
### 
    for column_id, column_name in enumerate(saved_df_columns):
        func_text.append(f'new_col_{column_id}_data = df._data[{df_columns_indx[column_name]}]')
        column_list.append((f'new_col_{column_id}_data', column_name))